pkg/preparer/setup.go

```diff
@@ -255,10 +266,11 @@ func (c *PreparerConfig) getOpts() (consul.Options, error) {
 	}
 
 	return consul.Options{
-		Address: c.ConsulAddress,
-		HTTPS:   c.ConsulHttps,
-		Token:   token,
-		Client:  client,
+		Address:  c.ConsulAddress,
+		HTTPS:    c.ConsulHttps,
+		Token:    token,
+		Client:   client,
+		WaitTime: c.ConsulConfig.WatchWaitTime,
```

Comment:
The default case (0s) would have some pretty profound effects on bandwidth, right? Unless consul uses 0 as a sentinel or something…